core/src/test/resources/util/test-application-multi.yml

```diff
@@ -0,0 +1,29 @@
+spring:
+  profiles:
+    active: dev,qa
+
+---
+
+spring:
+  profiles: dev
```

Comment:
not sure how this will come ??? you mean something like:

spring:
  profiles: dev, qa 

???